packages/react-scripts/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-scripts",
-  "version": "1.0.11",
+  "name": "@connected-home/react-scripts",
+  "version": "1.0.11.1",
```

Comment:
ah ok, you can't overwrite a published version. You can only delete them. prerelease versioning is your friend here.